website/docs/d/transit_sign.html.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: "vault"
+page_title: "Vault: vault_transit_sign data source"
+sidebar_current: "docs-vault-datasource-transit-sign"
+description: |-
+  Sign data using a Vault Transit signing key
+---
+
+# vault\_transit\_sign
+
+This is a data source which can be used to generate a signature using a Vault Transit key.
+
+## Example Usage
+
+```hcl
+data "vault_transit_sign" "test" {
+  backend     = "transit"
+  key         = "test"
+  input       = "aGVsbG8gd29ybGQ="
+}
+```
+
+## Argument Reference
+
+Each document configuration may have one or more `rule` blocks, which each accept the following arguments:
+
+* `name` - (Required) Specifies the name of the encryption key that was used to generate the signature or HMAC.
+
+* `backend` - (Required) The path the transit secret backend is mounted at, with no leading or trailing `/`.
+
+* `key_version` - (Optional) Specifies the version of the key to use for
+  signing. If not set, uses the latest version. Must be greater than or equal
+  to the key's `min_encryption_version`, if set.
+
+* `hash_algorithm` - (Optional) Specifies the hash algorithm to use for
+  supporting key types (notably, not including `ed25519` which specifies its
```

Comment:
we should probably remove this notably... sentence as it is no longer true for the ed25519ph stuff.